README.md

```diff
@@ -26,6 +26,25 @@ pip install git+https://github.com/mihaidusmanu/pycolmap
 
 This codebase includes external local features as git submodules – don't forget to pull submodules with `git submodule update --init --recursive`. Your local features are based on TensorFlow? No problem! See [below](#using-your-own-local-features-or-matcher) for the steps.
 
+
+#### Docker setup
+
+You can also use the docker image provided that already has colmap and all requirements installed. After building the image, you would just need to setup jupyter and run it through browser:
+
+```
+docker build -t hloc:latest .
+
+#run a container instance:
+
+docker run -it -p 8888:8888 hocl:latest
+
+#and start jupyter
+
+jupyter notebook --ip 0.0.0.0 --port 8888 --no-browser --allow-root
+```
```

Comment:
I suggest to remove comments and line breaks to make it compact - it shouldn't take much space in the README